src/catalog/postgres/builder.cpp

```diff
@@ -670,6 +676,31 @@ IndexSchema Builder::GetLanguageNameIndexSchema(db_oid_t db) {
   return schema;
 }
 
+Schema Builder::GetStatisticTableSchema() {
+  std::vector<Schema::Column> columns;
+
+  columns.emplace_back("starelid", type::TypeId::INTEGER, false,
+                       parser::ConstantValueExpression(type::TypeId::INTEGER));
+  columns.back().SetOid(PgStatistic::STARELID_COL_OID);
+
+  columns.emplace_back("staattnum", type::TypeId::INTEGER, false,
+                       parser::ConstantValueExpression(type::TypeId::INTEGER));
+  columns.back().SetOid(PgStatistic::STAATTNUM_COL_OID);
+
+//  columns.emplace_back("stanullfrac", type::TypeId::DECIMAL, false,
```

Comment:
Please remove dead code.

Reply:
@whoever is implementing the PR, Is this dead or not yet implemented? I think we'll need these columns as well as some binary array columns for topKs and histograms.

Reply:
@jkosh44 You'll probably be the one implementing it.